### ISSUE TITLE
Deleting a branch (and probably a tag) has no defined behavior

### ISSUE THREAD

**Issue by simensen** [dflydev/git-subsplit#6]
If a branch is deleted from a source repository the behavior is not defined. It is clear that the branch is not actually deleted from the splits but I'm not exactly sure what it is doing in that case.

**Comment by mnapoli** [comment]
Thanks @simensen for the awesome stuff, it will surely save me a headache.

If I understand correctly, branches from the source repo (the one containing all the other sub-repos) will be synced to sub-repos but never deleted right?

In the cases where you use this script, has it been a problem? Do you delete the branches manually in all the sub-repos?

**Comment by simensen** [comment]
Correct; I don't think that I've personally run into a case where a branch was synced into subtree split repositories that needed to be deleted but this would indeed need to be a manual process.
